source/stylesheets/refills/_badges.scss

```diff
@@ -1,36 +1,20 @@
-$badge-font-size: 0.75em !default;
-$medium-gray: #999 !default;
-$alert-color: #fff6bf !default;
-$error-color: #fbe3e4 !default;
-$notice-color: #e5edf8 !default;
-$success-color: #e6efc2 !default;
+$small-font-size: 0.75em;
+$badges: (
+  default: #999,
```

Comment:
It’s a limitation of using Sass maps and I think we thought they couldn't be added…however, per conversation above, looks like you can have a `!default` flag on maps, it just goes once at the very end.